index.js

```diff
@@ -42,13 +42,11 @@ myForm.addEventListener('submit', (e) => {
     localStorage.setItem('userData', serializedObject);
   }
 });
-// Add your Code below this line
-
 function fillData() {
   const desirializedObject = JSON.parse(localStorage.getItem('userData'));
   fullName.value = desirializedObject.fullName;
   emailInput.value = desirializedObject.emailInput;
   textArea.value = desirializedObject.textArea;
 }
```

Comment:
- [ ] From the project requirements, when a user changes the content of any input field, the data is saved to the local storage :smiley: . But in your project, the form must be submitted before it is saved to the localstorage. Please you can use the `change` event to get this done. example `inputElement.addEventListener('change', () => {})` :pray: